app/tracker/routes.js

```diff
@@ -145,7 +147,15 @@ angular.module(PKG.name + '.feature.tracker')
             controller: 'TrackerMetadataController',
             controllerAs: 'MetadataController',
             onEnter: function($stateParams) {
-              document.title = `${productName} | Search | ${$stateParams.entityId} | Summary`;
+              // Redirect to react page when the feature is turned on
+              if (window.CaskCommon.ThemeHelper.Theme.isMetadataInReact) {
+                let searchUrl = reactAppUrl.summary.replace('<namespace>', $stateParams.namespace);
```

Comment:
Is there any way to use router to avoid manual URL manipulation here?

Reply:
This is angular app's router. We need to construct the react url and redirect. Couldn't find other way.

Reply:
Arrrggghhh good point.